js/controller.js

```diff
@@ -326,10 +326,7 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
         this.state.pluginsClickElement.removeClass("showing");
         this.state.screenToShow = CONSTANTS.SCREEN.PLAYING_SCREEN;
         this.state.playerState = CONSTANTS.STATE.PLAYING;
-        if (Utils.isSafari()){
-          //Safari only can set cc when the video is playing, not before
```

Comment:
The issue was that this check was causing "this.setClosedCaptionsLanguage();" to only be done for Safari when it should be done for all browsers.